cli/src/cli/index.ts

```diff
@@ -185,76 +183,12 @@ export const runCli = async () => {
           },
         });
         cliResults.appName = appName;
-      }
-
-      const { language } = await inquirer.prompt<{ language: string }>({
-        name: "language",
-        type: "list",
-        message: "Will you be using JavaScript or TypeScript?",
-        choices: [
-          { name: "TypeScript", value: "typescript", short: "TypeScript" },
-          { name: "JavaScript", value: "javascript", short: "TypeScript" }, // Both options should have 'TypeScript' as the short value to improve UX and reduce confusion
-        ],
-        default: "typescript",
-      });
 
-      if (language === "javascript") {
-        logger.error("Wrong answer, using TypeScript instead...");
-      } else {
-        logger.success("Good choice! Using TypeScript!");
-      }
-
-      const { packages } = await inquirer.prompt<Pick<CliResults, "packages">>({
-        name: "packages",
-        type: "checkbox",
-        message: "Which packages would you like to enable?",
-        choices: availablePackages
-          .filter((pkg) => pkg !== "envVariables") // dont prompt for env-vars
-          .map((pkgName) => ({
-            name: pkgName,
-            checked: false,
-            // FIXME: TEMPORARY WARNING WHEN USING NODE 18. SEE ISSUE #59
-            disabled:
-              pkgName === "nextAuth" && process.versions.node.startsWith("18")
-                ? "Node.js version 18 is currently not compatible with Next-Auth."
-                : false,
-          })),
-      });
-
-      cliResults.packages = packages;
-
-      // Skip if noGit flag provided
-      if (!cliResults.flags.noGit) {
-        const { git } = await inquirer.prompt<{ git: boolean }>({
-          name: "git",
-          type: "confirm",
-          message: "Initialize a new git repository?",
-          default: true,
-        });
-        if (git) {
-          logger.success("Nice one! Initializing repository!");
-        } else {
-          cliResults.flags.noGit = true;
-          logger.info("Sounds good! You can come back and run git init later.");
-        }
-      }
-
-      if (!cliResults.flags.noInstall) {
-        const { runInstall } = await inquirer.prompt<{ runInstall: boolean }>({
-          name: "runInstall",
-          type: "confirm",
-          message: `Would you like us to run ${pkgManager} install?`,
-          default: true,
-        });
-
-        if (runInstall) {
-          logger.success("Alright. We'll install the dependencies for you!");
-        } else {
-          cliResults.flags.noInstall = true;
-          logger.info(
-            `No worries. You can run '${pkgManager} install' later to install the dependencies.`,
-          );
-        }
+        await askLanguage();
+        cliResults.packages = await askPackages();
+        if (!cliResults.flags.noGit) cliResults.flags.noGit = !(await askGit());
+        if (!cliResults.flags.noInstall)
+          cliResults.flags.noInstall = !(await askInstall());
```

Comment:
kind of hard to read when some break lines and some don't. In these cases it's better to just force it I think:

```ts
if (!cliResults.flags.noGit) {
  cliResults.flags.noGit = !(await promptGit());
}
if () {
  ...
}
```